packages/design/src/css/base.css

```
@@ -0,0 +1,30 @@
+/* src/css/base.css */
+@import url("./reset.css");
+
+@tailwind base;
+
+@import url("./tokens/index.css");
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)